### ISSUE TITLE
Updates of http://ftp.fau.de/pub/likwid/repos/

### ISSUE THREAD

**Issue by stevenvdb** [RRZE-HPC/likwid#498]
This question is not directly related to this Github repository, but I wanted to ask whether there are plans to update packages on http://ftp.fau.de/pub/likwid/repos? We are mirroring that server and we would like to install v5.2.2 rpms for CentOS_7 and CentOS_8.

**Comment by TomTheBear** [comment]
I forgot how to use the build service. It was never intended to provide the RPMs for all versions.

I'll try to add LIKWID 5.2.2.

**Comment by TomTheBear** [comment]
So, I triggered the build of LIKWID 5.2.2 and the RPMs are synced to [the ftp server](http://ftp.fau.de/pub/likwid/repos). There are already LIKWID 5.2.2 RPMs for CentOS7 & 8. I talked with the operators and it seems the service will be deprecated soon in favor for builds in containerized CI. So I'll add an `rpmbuild` Action in this repo. The RPMs will be available at the release page. I have such a setup already for a different project.
